stream_alert/athena_partition_refresh/main.py

```diff
@@ -418,13 +434,24 @@ def __init__(self, config):
 
         self.setup()
 
+    @property
+    def queue_name(self):
+        """Return the name of the sqs queue to use. This can be overridden in the config"""
+        queue = self.config['lambda']['athena_partition_refresh_config'].get('queue_name', '')
+        queue = queue.replace(' ', '') # strip any spaces which are invalid queue names
+        if queue == '':
```

Comment:
`if not queue:`

Reply:
This code will probably change but great suggestion!